stac-extension/README.md

```diff
@@ -0,0 +1,95 @@
+# Humanitarian OpenStreetMap Team OpenAerialMap Extension Specification
+
+- **Title:** Humanitarian OpenStreetMap Team OpenAerialMap (OAM) Extension
+- **Identifier:** <https://raw.githubusercontent.com/hotosm/stactools-hotosm/refs/heads/main/stac-extension/json-schema/schema.json>
+- **Field Name Prefix:** oam
+- **Scope:** Item
+- **Extension [Maturity Classification](https://github.com/radiantearth/stac-spec/tree/master/extensions/README.md#extension-maturity):** Proposal
+- **Owner**: @ceholden @gadomski
+
+This extension documents metadata used by the Humanitarian OpenStreetMap Team (HOT)'s OpenAerialMap (OAM) project.
+It builds on common STAC metadata by defining some optional common metadata as required attributes and defines
+the expected values for common metadata.
+
+- Examples:
+  - [Item example](./examples/item.json): Shows the basic usage of the extension in a STAC Item
+- [JSON Schema](./json-schema/schema.json)
+- [Changelog](./CHANGELOG.md)
+
+## Fields
+
+The fields in the table below can be used in these parts of STAC documents:
+
+- [ ] Catalogs
+- [ ] Collections
+- [x] Item Properties (incl. Summaries in Collections)
+- [ ] Assets (for both Collections and Items, incl. Item Asset Definitions in Collections)
+- [ ] Links
+- [x] Provider
+
+| Field Name        | Type   | Description                                                                                          |
+| ----------------- | ------ | ---------------------------------------------------------------------------------------------------- |
+| provider          | string | **REQUIRED**. The imagery data provider. The provider must also be included in the "provider" field. |
+| gsd               | number | **REQUIRED**. The Ground Sampling Distance                                                           |
+| oam:platform_type | string | **REQUIRED**. The platform type (kite, balloon, UAV, airplane, satellite)                            |
+| license           | string | If provided for STAC Items, must be a Creative Commons (CC) license.                                 |
+
+### Additional Field Information
+
+#### oam:platform_type
+
+The type of the observation platform used to acquire the imagery. The platform type may be,
+
+- `kite`
+- `balloon`
+- `UAV`
```

Comment:
These looks like slugs, so maybe should be all lowercase (even though this is a acronym?

```suggestion
- `uav`
```